This & seems wrong, but the static_cast to QItemSelectionModel::SelectionFlags
is required anyway to make gcc6 happy.

--- a/src/plugins/accessible/widgets/itemviews.cpp	2015-05-07 16:14:48.000000000 +0200
+++ b/src/plugins/accessible/widgets/itemviews.cpp	2016-09-11 23:17:33.960195447 +0200
@@ -393,7 +393,7 @@
     QModelIndex index = view()->model()->index(0, column, view()->rootIndex());
     if (!index.isValid() || view()->selectionMode() & QAbstractItemView::NoSelection)
         return false;
-    view()->selectionModel()->select(index, QItemSelectionModel::Columns & QItemSelectionModel::Deselect);
+    view()->selectionModel()->select(index, static_cast<QItemSelectionModel::SelectionFlags>(QItemSelectionModel::Columns & QItemSelectionModel::Deselect));
     return true;
 }
 
